src/kOS.Safe/Execution/CPU.cs

```diff
@@ -155,9 +159,23 @@ public void Boot()
                 if (file == null)
                 {
                     SafeHouse.Logger.Log(string.Format("Boot file \"{0}\" is missing, skipping boot script", path));
+
+                    shared.Screen?.Print(string.Format(
+                        " \n" +
+                        "Could not boot from {0}\n" +
+                        "The file is missing\n" +
```

Comment:
Missing period in the sentence?

Reply:
Where would you like a period?

Reply:
The statement "The file is missing" is a separate sentence that isn't continued on the next line.  (Instead a new sentence starts on the next line).  So, yeah, it feels like a period is needed before the ``\n``.

Reply:
Surprised you don't want it after `{0}` as that is clearly also *separate sentence that isn't continued on the next line. Does not compute*

Reply:
It's both.  For some reason when github showed these comments in the PR thread, it was under line 165 when it shows it under line 166 on the longer diff output, so I didn't see the second sentence on line 166 when replying.

Reply:
Does those dot-fixes make it any closer to merging? I don't think so. I will consider creating that minimalistic PR tomorrow... and maybe never. I have my working version and it does not feel like I am of any help here.